File: curdapp/src/main/java/com/springboot/curdapp/service/Student.java
package com.springboot.curdapp.service;

import javax.persistence.*;

@Entity
@Table(name = "student")
public class Student {

    @Id
    private int id;
    private String name;
    private long rollno;
    private String course;

    public Student(int id, String name, long rollno, String course) {
        this.id = id;
        this.name = name;
        this.rollno = rollno;
        this.course = course;
    }

    public Student() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getRollno() {
        return rollno;
    }

    public void setRollno(long rollno) {
        this.rollno = rollno;
    }

    public String getCourse() {
        return course;
    }

    public void setCourse(String course) {
        this.course = course;
    }

    @Override
    public String toString() {
        return "Student{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", rollno=" + rollno +
                ", course='" + course + '\'' +
                '}';
    }
}

File: angular src/app/Student.ts
export default class Student{
     id:number;
     name:string;
     rollno:number;
     course:string;

    constructor(id:number,name:string,rollno:number,course:string)
    {
       this.id=id;
        this.name=name;
        this.rollno=rollno;
        this.course=course;
    }
    static getRandomNumber():number{

        return Math.ceil(Math.random()*1000);
    }


}